src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -20,5 +21,28 @@ public IObservable<IPullRequestModel> CreatePullRequest(IRepositoryHost host, IS
 
             return host.ModelService.CreatePullRequest(repository, title, body, source, target);
         }
+
+        public string GetPullRequestTemplate(ISimpleRepositoryModel repository)
+        {
+            Extensions.Guard.ArgumentNotNull(repository, nameof(repository));
+
+            var paths = new[]
+            {
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE.md"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE.md"),
```

Comment:
Sure, if you think that'd be better. However think I prefer to have everything in one place so it can be read more easily. Let me know if you have strong feelings either way.

I can definitely reorder though.